components/gallery/GalleryItemAction/GalleryItemActionType/GalleryItemOffer.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <div class="flex justify-end gallery-item-offer mr-[55px]">
+    <NeoButton
+      :label="$t('transaction.offer')"
+      variant="k-blue"
+      size="large"
+      fixed-width
+      @click="openOfferModal"
+    />
```

Comment:
@exezbcz this looks weird to me

<img src=https://github.com/user-attachments/assets/d0f9a6a6-ccec-46be-88e6-11ea0e4f67e4 width=400 />

maybe 

<img src=https://github.com/user-attachments/assets/844a35ea-e40b-4d90-8a9a-99fd2d7a8fa5 width=400 />

Reply:
yes, that was the design in the 2022

Reply:
updated